packages/ui/jumbotron/src/lib/jumbotron/jumbotron.component.spec.ts

```
@@ -5,8 +5,8 @@ import { By } from '@angular/platform-browser';
 
 @Component({
   template: `<lib-jumbotron>trasd</lib-jumbotron>`,
-  imports:[JumbotronComponent],
-  selector:'lib-acncu'
+  imports: [JumbotronComponent],
+  selector: 'lib-acncu',
 })
 export class TestJumbotronComponent {}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)